### PR TITLE
SRVCOM-2141  Revert to earlier version of Installing Operator docs

### PR DESCRIPTION
**Pull request by gabriel-rh**
Version(s):
4.8+

Issue:
https://issues.redhat.com/browse/SRVCOM-2141

Link to docs preview:
https://52773--docspreview.netlify.app/openshift-enterprise/latest/serverless/install/install-serverless-operator.html

QE review:
- [ ] QE has approved this change.

### CODE REVIEW

File: modules/serverless-cluster-sizing-req.adoc [hunk 1]
```diff
@@ -6,13 +6,14 @@
...
+To install and use {ServerlessProductName}, the {product-title} cluster must be sized correctly. 
 
 [NOTE]
 ====
 The following requirements relate only to the pool of worker machines of the {product-title} cluster. Control plane nodes are not used for general scheduling and are omitted from the requirements.
 ====
 
-By default, each pod requests approximately 400m of CPU, so the minimum requirements are based on this value. Lowering the actual CPU request of applications can increase the number of possible replicas.
+The minimum requirement to use {ServerlessProductName} is a cluster with 10 CPUs and 40GB memory.
+By default, each pod requests ~400m of CPU, so the minimum requirements are based on this value.
```
Comment (skonto) [inline modules/serverless-cluster-sizing-req.adoc:17]: Should not we have numbers for installation reqs per component (SRVCOM-1560 has that)? There are some cases that people dont want to install all components.

**Comment by skonto** [comment]
LGTM

**Review by jc-berger — APPROVED** [review]
/lgtm

**Pull request merged**